lib/logstash/inputs/syslog.rb

```diff
@@ -30,7 +30,7 @@ class LogStash::Inputs::Syslog < LogStash::Inputs::Base
   default :codec, "plain"
 
   # The address to listen on.
-  config :host, :validate => :string, :default => "0.0.0.0"
+  config :host, :validate => :string, :default => "::"
```

Comment:
do not think we should change the plugin's host default to IPv6 just because we handle IPv6 support
this should stay a plain old IPv4

Reply:
Hi, I've removed the default. 

However, normally the IPv6 listen on all implies that the it also listens on 0.0.0.0 (unless somebody changes this system behaviour on their system, if so, I reckon that these people would also configure the address to listen on ipv4).